lib/fastlane/fast_file.rb

```diff
@@ -105,7 +105,7 @@ def platform(platform_name, &block)
 
       self.current_platform = platform_name
 
-      block.call
+      yield
```

Comment:
Does that really work? I mean we have the block inside the `block` variable, right?